src/Array.jl

```diff
@@ -119,6 +120,27 @@ function Matrix{T}(g::Gate{<:Control}) where {T}
     return M
 end
 
+function Base.rand(ElType::Type, ::Type{SU{N}}, lanes::NTuple{M, Int}; seed::Union{Int, Nothing}=nothing) where {N, M}
+    if seed !== nothing
+        Random.seed!(seed)
+    end
```

Comment:
Don't add seeding like this. If a user wants to set the seed, let the user import `Random` and call `Random.seed!`, not us.

Also, it's very strange to have 2 `::Type` arguments.

```suggestion
function Base.rand(::Type{SU{N}}, lanes::NTuple{M, Int}; eltype::Type = ComplexF64) where {N, M}
```

Reply:
Okey!